test/helpers/hotwire_combobox/helper_test.rb

```diff
@@ -138,6 +138,15 @@ class HotwireCombobox::HelperTest < ApplicationViewTestCase
     assert_attrs html, tag_name: "turbo-frame", src: "/foo?bar=baz&qux=quux&page=2&format=turbo_stream"
   end
 
+  test "hw_paginated_combobox_options includes array params" do
+    request.expects(:path).returns("/foo")
+    request.expects(:query_parameters).returns({ ary: [ 1, 2 ] }.with_indifferent_access)
+
+    html = hw_paginated_combobox_options [], next_page: 2
+
+    assert_attrs html, tag_name: "turbo-frame", src: "/foo?ary=1&ary=2&page=2&format=turbo_stream"
```

Comment:
As far as I know, Rails prefers the `ary[]=1&ary[]=2...` convention for array/nested params. I believe we get that for free with Rack's `build_nested_query`. Are you depending on the `ary=1&ary=2` convention instead?

I think to get a passing test using that notation we'd need to unescape the url encoding in the result. Something like:

`assert_attrs CGI.unescape(html), tag_name: "turbo-frame", src: "/foo?ary[]=1&ary[]=2&page=2&format=turbo_stream"`